fe/lib/components/LocationLookup/LocationLookup.stories.tsx

```diff
@@ -1,36 +1,20 @@
-import 'braid-design-system/reset';
+import type { Meta, StoryObj } from 'sku/@storybook/react';
 
-import React, { type ComponentProps } from 'react';
+import { defaultArgTypes } from '../../../.storybook/preview';
 
-import {
-  type BraidArgs,
-  defaultArgTypes,
-  defaultArgs,
-} from '../../storybook/controls';
-import { BraidStorybookProvider } from '../../storybook/decorators';
-
-import { LocationLookup as Component } from './LocationLookup';
-import { MockLocationLookup } from './LocationLookup.mock';
+import { MockLocationLookup as Component } from './LocationLookup.mock';
 
 export default {
+  title: 'Job Posting/Locations/LocationLookup',
```

Comment:
Do we know if this migration affects the Storybook URLs?

If so we definitely need a release and have to update the manual fragments in the `.mock.tsx` components:

https://github.com/seek-oss/wingman/blob/9332027d88f56be48d1fd3288bb96db0d1ec23a0/fe/lib/components/LocationLookup/LocationLookup.mock.tsx#L32

Reply:
The URLs should be unaffected by this migration 👍